### ISSUE TITLE
Package update: FFTW 3.3.6-pl2

### ISSUE THREAD

**Issue by Barbouze** [KXStudio/Repository#113]
Hi!
ambiX needs fftw 3.3.6-pl2 to be built and KXStudio version is 3.3.5, could you please consider updating it?

**Comment by falkTX** [comment]
the fftw in the kx repos is used static and only useful for repo builds.
are you using that package?
it's already 3.3.5 from git, so basically 3.3.6 in the end.

**Comment by Barbouze** [comment]
Well, I tried to ^^'
As the default fftw library provided by Mint 18.2 is 3.3.4-2, I tried to compile ambix with this static library but had problems at linking time anyway. I will go back to RTFM and upgrade my skills on compiling ;)

FYI, there are bugfixes in 3.3.6:
- "The fftw_make_planner_thread_safe() API introduced in 3.3.5 didn't work, and this 3.3.6 fixes it. Sorry about that. " 
- "FFTW 3.3.6 had the wrong libtool version number, and generated shared libraries of the form libfftw3.so.2.6.6 instead of libfftw3.so.3.*. "
